autopprof_test.go

```diff
@@ -858,13 +1020,13 @@ func BenchmarkLightJob(b *testing.B) {
 
 func BenchmarkLightJobWithWatchCPUUsage(b *testing.B) {
```

Comment:
i think we need to benckmark goroutine watcher too.

Reply:
I artificially created a lot of goroutines and ran a benchmark test. This is my first time doing a test like this. What should I fix?
```
BenchmarkLightAsyncJob-5                                  376760             32056 ns/op            7040 B/op        352 allocs/op
BenchmarkLightAsyncJobWithWatchGoroutineCount-5           350994             33689 ns/op            7040 B/op        351 allocs/op
BenchmarkHeavyAsyncJob-5                                     414          29095456 ns/op         6001969 B/op     300096 allocs/op
BenchmarkHeavyAsyncJobWithWatchGoroutineCount-5              403          29474789 ns/op         5972177 B/op     298607 allocs/op
```